### DISCUSSION TITLE
FUNCTIONS_WORKER_RUNTIME change - explain this properly please

### DISCUSSION THREAD

**Discussion by TWolversonReply** [Azure/logicapps#1210] [Help Needed]
A note has appeared in the [documentation](https://learn.microsoft.com/en-us/azure/logic-apps/edit-app-settings-host-settings?tabs=azure-portal#reference-for-app-settings---localsettingsjson)

> Note: This setting's value was previously set to node, but now the required value is dotnet for all new and existing deployed Standard logic apps. This change shouldn't affect your workflow's runtime, so everything should work the same way as before.

Why? Please be clearer and more up-front about the reason for this change and what impact, if any, it has. Do I need to update templates? What happens if I don't?

Additionally, this begs the question: If it doesn't affect my workflow's runtime, then why is it required?

**Comment by FlipFlop17** [comment]
We just realized that in our arm template for our  Logic app we have set it to 'node' but after the app is deployed the value for FUNCTIONS_WORKER_RUNTIME is set to 'dotnet'.
Are they defaulting to 'dotnet' always now ??! Very curious

**Reply by karl-lonnberg** [reply]
What does you Stack setting looks like? This is from a fresh deployment after I changed runtime to "dotnet". (I did not specified .NET version in the iac):
![image](https://github.com/user-attachments/assets/59375d85-210b-4675-8718-c3307dd15a69)

I believe this is a bad error message as dotnet-isolated is deprecated from what I've heard.

**Comment by DieterGobeyn** [comment]
With this upgrade in mind, what impacts should we anticipate when moving from .NET 6 to .NET 8? Specifically, will the host instance itself be updated in this process? Our Logic Apps don’t contain any custom .NET code, so I’d like to better understand if and how this change could affect us.

![image](https://github.com/user-attachments/assets/5eee20f4-9c61-4f50-b33d-67dc1f403c3a)
